system/scripts/zone/ais/sadhu_dummy.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Melia.Shared.Game.Const;
+using Melia.Zone;
+using Melia.Zone.Network;
+using Melia.Zone.Scripting;
+using Melia.Zone.Scripting.AI;
+using Melia.Zone.Skills;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.Characters;
+using Melia.Zone.World.Actors.Characters.Components;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+
+[Ai("SadhuDummy")]
+public class SadhuDummyAiScript : AiScript
+{
+	private const int MaxChaseDistance = 50;
+	private const int MaxMasterDistance = 80;
+
+	ICombatEntity target;
+
+	protected override void Setup()
+	{
+		SetViewDistance(200);
+
+		SetTendency(TendencyType.Aggressive);
+		HatesFaction(FactionType.Peaceful);
+		HatesFaction(FactionType.Pet);
+		HatesFaction(FactionType.Monster);
+		HatesFaction(FactionType.Neutral);
+		HatesFaction(FactionType.Summon);
+
+		During("Idle", CheckEnemies);
+		During("Attack", CheckTarget);
+		During("Attack", CheckMaster);
+	}
+
+	protected override void Root()
+	{
+		StartRoutine("Idle", Idle());
+	}
+
+	protected IEnumerable Idle()
+	{
+		var movement = this.Entity.Components.Get<MovementComponent>();
+
+		movement.SetMoveSpeedType(MoveSpeedType.Run);
+		
+		if (this.Entity is Character entityCharacter && entityCharacter is DummyCharacter dummyCharacter)
+		{
+			SetFixedMoveSpeed(55);
+			entityCharacter.Properties.Modify(PropertyName.MSPD_BM, 55);
+			Send.ZC_MSPD(dummyCharacter.Owner, entityCharacter);
+		}
+		
+		var master = GetMaster();
+		if (master != null && !InRangeOf(master, MaxMasterDistance))
+		{
+			yield return Follow(master, 25, true, false);
+			yield break;
+		}
+
+		yield return Wait(250, 500);
+
+		SwitchRandom();
+
+		if (Case(80))
+		{
+			yield return MoveRandom();
+		}
+	}
+
+	protected IEnumerable Attack()
+	{		
+		// Remove the dummy character if the master is gone
```

Comment:
You should not send raw packets from AI scripts unless there's absolutely no other way to accomplish what you need to do, because scripts, by their nature, should be simple to understand for users who are less experienced.

A cleaner solution would be to call some type of despawn method, which then internally figures out how to remove the dummy or whatever other actor the AI might be controlling.